sample/index.html

```diff
@@ -33,15 +33,20 @@ <h2>Formula rendering from markup</h2>
 		<script src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.js"></script>
     	<script src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/contrib/auto-render.min.js"></script>
 		<script src="../build/ckeditor.js"></script>
+		<script src="../build/translations/en.js"></script>
+		<script src="../build/translations/es.js"></script>
+		<script src="../build/translations/uk.js"></script>
 		<script>
 			console.log(
```

Comment:
Do we want to keep the console.log?

Reply:
this index.html is only for local preview of the built editor. let's keep it :)